app/src/main/res/xml/prediction_fragment_scene.xml

```diff
@@ -0,0 +1,59 @@
+<?xml version="1.0" encoding="utf-8"?>
+<MotionScene xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:motion="http://schemas.android.com/apk/res-auto">
+
+    <ConstraintSet android:id="@+id/start">
+
+        <Constraint
+            android:id="@+id/button_start_predict"
+            android:layout_width="100dp"
+            android:layout_height="100dp"
+            motion:layout_constraintBottom_toBottomOf="parent"
+            motion:layout_constraintEnd_toEndOf="@+id/guideline_centerLeft_predictionFragment"
+            motion:layout_constraintTop_toTopOf="parent"
+            motion:shapeAppearanceOverlay="@style/ShapeAppearanceOverlay.MyApp.Button.Circle"
+
+            />
+
+        <Constraint
+            android:id="@+id/tv_predict_prediction"
+            android:layout_width="200dp"
+            android:layout_height="wrap_content"
+            android:visibility="gone"
+            motion:layout_constraintBottom_toBottomOf="parent"
+            motion:layout_constraintEnd_toEndOf="@id/timer_predict_predict"
+            motion:layout_constraintStart_toStartOf="@id/timer_predict_predict"
+            motion:layout_constraintTop_toBottomOf="@id/timer_predict_predict" />
+    </ConstraintSet>
+
+    <ConstraintSet android:id="@+id/end">
+
+        <Constraint
+            android:id="@+id/button_start_predict"
+            android:layout_width="100dp"
+            android:layout_height="100dp"
+            motion:layout_constraintBottom_toBottomOf="parent"
+            motion:layout_constraintEnd_toEndOf="@+id/guideline_left_predictionFragment"
+            motion:layout_constraintTop_toTopOf="parent"
+            motion:shapeAppearanceOverlay="@style/ShapeAppearanceOverlay.MyApp.Button.Circle"
+            />
+
+        <Constraint
+            android:id="@+id/tv_predict_prediction"
+            android:layout_width="200dp"
+            android:layout_height="wrap_content"
+            android:visibility="visible"
+            motion:layout_constraintBottom_toBottomOf="parent"
+            motion:layout_constraintEnd_toEndOf="@id/timer_predict_predict"
+            motion:layout_constraintStart_toStartOf="@id/timer_predict_predict"
+            motion:layout_constraintTop_toBottomOf="@id/timer_predict_predict"
+           />
+    </ConstraintSet>
+
+    <Transition
+        motion:constraintSetEnd="@id/end"
+        motion:constraintSetStart="@+id/start"
+        motion:duration="500">
+        <!--        <OnClick motion:targetId="@+id/button_start_predict"/>-->
+    </Transition>
```

Comment:
Comment should be removed (Sorry, meiner!)

Reply:
its gone now